notebooks/demo/nx_cugraph_demo.ipynb

```diff
@@ -0,0 +1,672 @@
+      "NX: (5, 0.029987), CG: (5, 0.029987)\n",
+      "NX: (6, 0.029987), CG: (6, 0.029987)\n",
+      "NX: (27, 0.022333), CG: (27, 0.022333)\n",
+      "NX: (23, 0.017614), CG: (23, 0.017614)\n",
+      "NX: (30, 0.014412), CG: (30, 0.014412)\n",
+      "NX: (3, 0.011909), CG: (3, 0.011909)\n",
+      "NX: (25, 0.003840), CG: (25, 0.003840)\n",
+      "NX: (29, 0.002922), CG: (29, 0.002922)\n",
+      "NX: (24, 0.002210), CG: (24, 0.002210)\n",
+      "NX: (28, 0.001795), CG: (28, 0.001795)\n",
+      "NX: (9, 0.000848), CG: (9, 0.000848)\n",
+      "NX: (4, 0.000631), CG: (4, 0.000631)\n",
+      "NX: (10, 0.000631), CG: (10, 0.000631)\n",
+      "NX: (7, 0.000000), CG: (7, 0.000000)\n",
+      "NX: (11, 0.000000), CG: (11, 0.000000)\n",
+      "NX: (12, 0.000000), CG: (12, 0.000000)\n",
+      "NX: (14, 0.000000), CG: (14, 0.000000)\n",
+      "NX: (15, 0.000000), CG: (15, 0.000000)\n",
+      "NX: (16, 0.000000), CG: (16, 0.000000)\n",
+      "NX: (17, 0.000000), CG: (17, 0.000000)\n",
+      "NX: (18, 0.000000), CG: (18, 0.000000)\n",
+      "NX: (20, 0.000000), CG: (20, 0.000000)\n",
+      "NX: (21, 0.000000), CG: (21, 0.000000)\n",
+      "NX: (22, 0.000000), CG: (22, 0.000000)\n",
+      "NX: (26, 0.000000), CG: (26, 0.000000)\n"
+     ]
+    }
+   ],
+   "source": [
+    "# The lists contain tuples of (vertex ID, betweenness_centrality score),\n",
+    "# sorted based on the score.\n",
+    "nx_sorted = sorted(karate_nx_bc_results.items(), key=lambda t:t[1], reverse=True)\n",
+    "cg_sorted = sorted(karate_cg_bc_results.items(), key=lambda t:t[1], reverse=True)\n",
+    "\n",
+    "for i in range(len(nx_sorted)):\n",
+    "    print(\"NX: (%d, %.6f), CG: (%d, %.6f)\" % (nx_sorted[i] + cg_sorted[i]))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "Here we can see that the results match exactly as expected.  \n",
+    "\n",
+    "For larger graphs, results are harder to compare given that `betweenness_centrality` is an approximation algorithm influenced by the random selection of paths used to compute the betweenness_centrality score of each vertex.  The argument `k` is used for limiting the number of paths used in the computation, since using every path for every vertex would be prohibitively expensive for large graphs.  For small graphs, `k` need not be specified, which allows `betweenness_centrality` to use all paths for all vertices and makes for an easier comparison."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### `betweenness_centrality` on larger graphs - The U.S. Patent Citation Network<sup>1</sup>\n",
+    "\n",
+    "The U.S. Patent Citation Network dataset is much larger with over 3.7M nodes and over 16.5M edges, and demonstrates how `nx-cugraph` enables NetworkX to run `betweenness_centrality` on graphs this large (and larger) in seconds instead of minutes.\n",
```

Comment:
```suggestion
    "The U.S. Patent Citation Network dataset is much larger with over 3.7M nodes and over 16.5M edges and demonstrates how `nx-cugraph` enables NetworkX to run `betweenness_centrality` on graphs this large (and larger) in seconds instead of minutes.\n",
```